core-java-modules/core-java-perf/src/test/java/com/baeldung/jmxshell/custom/JmxInvokerLiveTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.jmxshell.custom;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
```

Comment:
I think checkstyle needs an empty blank line between static and non-static imports.